src/zos_jobs/zowe/zos_jobs_for_zowe_sdk/jobs.py

```diff
@@ -80,12 +80,10 @@ def cancel_job(self, jobname: str, jobid: str, modify_version="2.0"):
         job_url = "{}/{}".format(jobname, jobid)
         request_url = "{}{}".format(self.request_endpoint, job_url)
         custom_args["url"] = request_url
-        custom_args["json"] = {
-            "request": "cancel",
-            "version": modify_version
-        }
-
-        response_json = self.request_handler.perform_request("PUT", custom_args, expected_code=[202, 200])
+        custom_args["json"] = {"request": "cancel", "version": modifyVersion}
+        response_json = self.request_handler.perform_request("PUT", custom_args, expected_code = [202])
+        custom_args["json"] = {"request": "cancel"}
+        response_json = self.request_handler.perform_request("PUT", custom_args, expected_code=[202])
```

Comment:
Please remove the duplicate REST calls and use `202, 200` for the `expected_code`.